packages/protocol/contracts/common/FeeCurrencyWhitelist.sol

```diff
@@ -2,14 +2,14 @@ pragma solidity ^0.5.3;
 
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
-import "./interfaces/IGasCurrencyWhitelist.sol";
+import "./interfaces/IFeeCurrencyWhitelist.sol";
```

Comment:
nit: In places where it isn't obvious (e.g. tx params) should we call this `TxFeeCurrencyWhitelist`?

Don't feel strongly

Reply:
I can see it both ways. I think my slight preference is toward `FeeCurrencyWhitelist`